stk/calculators/energy/energy_calculators.py

```diff
@@ -377,3 +382,847 @@ def energy(self, mol, conformer=-1):
         rdkit.GetSSSR(mol.mol)
         ff = rdkit.UFFGetMoleculeForceField(mol.mol, confId=conformer)
         return ff.CalcEnergy()
+
+
+class XTBEnergy(EnergyCalculator):
+    """
+    Uses GFN-xTB to calculate energies and other properties of molecules.
+
+    Notes
+    -----
+    When running :meth:`energy`, this calculator changes the
+    present working directory with :func:`os.chdir`. The original
+    working directory will be restored even if an error is raised so
+    unless multi-threading is being used this implementation detail
+    should not matter.
+
+    If multi-threading is being used an error could occur if two
+    different threads need to know about the current working directory
+    as this :class:`.EnergyCalculator` can change it from under them.
+
+    Note that this does not have any impact on multi-processing,
+    which should always be safe.
+
+    Documentation for xTB available:
+    https://xtb-docs.readthedocs.io/en/latest/setup.html
+
+    Attributes
+    ----------
+    xtb_path : :class:`str`
+        The path to the xTB executable.
+
+    gfn_version : :class:`str`
+        Parameterization of GFN to use in xTB.
+        For details:
+            https://xtb-docs.readthedocs.io/en/latest/basics.html
+
+    output_dir : :class:`str`
+        The name of the directory into which files generated during
+        the optimization are written, if ``None`` then
+        :func:`uuid.uuid4` is used.
+
+    num_cores : :class:`int`
+        The number of cores for xTB to use. Requires appropriate setup
+        of xTB by user.
+
+    etemp : :class:`int`
```

Comment:
I would prefer if this was called electronic_temperature. stk uses temperature instead of temp throughout.